### PR TITLE
GitHab Pages 

### PR DESCRIPTION
**Pull request by mrkamin**
What does this PR do?
This PR is raised to use GitHub pages which allows you to host (deploy) a webpage from your repository.

Description of the task to be completed 👎
 - Add the HTML for the structure.
 - Add the CSS file for the styling.
 - Use the linter tool that analyzes my source code to flag programming errors, bugs, stylistic errors, and suspicious constructs. 💯
 - Use Figma a collaborative UI design tool built in the browser.
 - Use Flexbox a layout tool in CSS that helps you easily position HTML elements on the page.
 - This PR is raised to use GitHub pages which allows you to host (deploy) a webpage from your repository, for more information about how to use GitHub pages [Read here](https://www.w3schools.com/git/git_remote_pages.asp?remote=github) 🤩 if you want to watch a video about how to use GitHub pages instead of reading [Click here](https://www.youtube.com/watch?v=QyFcl_Fba-k&t=251s) ☺️.
 This is my portfolio hosted on GitHub pages, so if you want to check my portfolio online, please 😊 [Click here](https://mrkamin.github.io/)
How should it be tested
Clone the repository

git clone git@github.com:mrkamin/mrkamin.github.io.git
Navigate to the folder

cd portfolio
Checkout the branch

git checkout github_pages
Thanks for reviewing my PR 😊

### CODE REVIEW

**Comment by mrkamin** [comment]
Add live Web Site Url

File: README.md [hunk 1]
```diff
@@ -1,14 +1,44 @@
...
+- Flexbox
 - Figma
--
+- Formspree service
+- (RWD) responsive web design using media query
+- Animation and Transition
+
+
+
+## Live Demo (if available)
```
Comment (mgmediaweb) [inline README.md]: Kindly, I suggest you put the link to your page displayed here.

**Review by mgmediaweb — CHANGES_REQUESTED** [review]
Hi @mrkamin , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

work2

## Highlights 💯 
- Professional-looking readme file. ✔️ 
- Clean and tidy code. ✔️ 
- Good job following the design. ✔️ 

## Required Changes ♻️

*** Kindly, I suggest you create a more descriptive Pull Request of the work you are presenting, this will help the reviewer to better understand your work.

*** Kindly, the latest version of your deployment produces an error, I suggest you adjust it and send this same PR again.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (mrkamin) [reply README.md]: > Kindly, I suggest you put the link to your page displayed here.

@mgmediaweb  could you please tell me a little clear in which line I should put and which link

Comment (mrkamin) [reply README.md]: > Kindly, I suggest you put the link to your page displayed here.

@mgmediaweb you mentioned that my latest deployed version produces errors kindly please make it a little clear what kind of errors it has and could you please share a screenshot of the errors it is not obvious to me thanks a lot

Comment (mgmediaweb) [reply README.md]: Line 19, you need remove "if available" and bellow put the URL to your deployed page

Comment (mrkamin) [reply README.md]: @mgmediaweb  kindly could you please tell me more about the error which pordueced

**Comment by mrkamin** [comment]
add  description to pull request

File: README.md [hunk 2]
```diff
@@ -1,14 +1,44 @@
...
+
+- Gain access to this awesom project by :
+- heading over to "https://github.com/mrkamin/mrkamin.github.io" to view the Github repo.
+- Forking it to your Github account (You can as well "clone" it to your local machine )
+And there you have it all running in your local machine 😎
+
+### Links
+
+- Solution URL: [https://github.com/mrkamin/mrkamin.github.io]
+- Live Site URL: [https://mrkamin.github.io/]
```
Comment (1GORDON) [inline README.md:29]: Great job @mrkamin 🤝

- [ ] [Optional] Kindly use [markdown syntax](https://www.markdownguide.org/basic-syntax/) to organize the content of the readme file.

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @mrkamin ✋

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Followed Gitflow ✔️ 
- Good commit messages ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

## Optional suggestions

- [ ] Kindly make sure the mobile version of your design works properly as well. The deployed link seems to contain only the desktop version.
- [ ] Kindly make sure your deployed version is also responsive.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**